assets/populate.py

```diff
@@ -20,16 +21,16 @@ def getOrCreateAlbum(name, artistId):
 	return out["id"]
 
 def getOrCreateGenre(names):
-    ids = []
```

Comment:
oops :)

Reply:
tbh i didnt code that line

Reply:
i just changed space indent to tab indent